### PR TITLE
feat: verifier profile management REST API + stubs for issuer API

### PR DESCRIPTION
**Pull request by aholovko**
Signed-off-by: Andrii Holovko <andriy.holovko@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,13 +34,14 @@ all: checks unit-test bdd-test
 .PHONY: checks
 checks: license lint open-api-spec
 
-.PHONY: mocks
-mocks:
+.PHONY: generate
+generate:
 	@GOBIN=$(GOBIN_PATH) go install github.com/golang/mock/mockgen@$(MOCK_VERSION)
+	@GOBIN=$(GOBIN_PATH) go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@latest
 	@go generate ./...
```
Comment (sudeshrshetty) [inline Makefile:41]: why we have common generate command for mocks & open apis here? they are dependent on each other?

**Review by sudeshrshetty — APPROVED** [review]

**Review by fqutishat — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (aholovko) [reply Makefile:41]: "generate" is a global command, we can't differentiate one type of "generate" from another
in our case, they come hand-in-hand, ie.
```
//go:generate oapi-codegen --config=openapi.cfg.yaml ../../../../api/openapi.yaml
//go:generate mockgen -destination controller_mocks_test.go ...
```

Comment (sudeshrshetty) [reply Makefile:41]: if some one wants to generate open api spec, why he has to wait for mocks to get generated?

I am fine with having common `generate` command which generates everything (grouping other sub generate commands), but I would suggest having separate for mock, open api...and common generate command as umbrella command

Comment (aholovko) [reply Makefile:41]: makes sense, I will try to come up with some solution in the upcoming PRs cc@sudeshrshetty

**Pull request merged**